src/main/java/org/quantumbadger/redreader/reddit/url/RedditURLParser.java

```diff
@@ -97,45 +107,46 @@ public static RedditURL parse(Uri uri) {
 	public static RedditURL parseProbableCommentListing(Uri uri) {
 
 		RedditURL matchURL = parse(uri);
-		if(matchURL != null) return matchURL;
+		if (matchURL != null) return matchURL;
 
 		return new UnknownCommentListURL(uri);
 	}
 
 	public static RedditURL parseProbablePostListing(Uri uri) {
 
 		RedditURL matchURL = parse(uri);
-		if(matchURL != null) return matchURL;
+		if (matchURL != null) return matchURL;
 
 		return new UnknownPostListURL(uri);
 	}
 
 	public static abstract class RedditURL {
 		public abstract Uri generateJsonUri();
-		public abstract PathType pathType();
+
+		public abstract @PathType int pathType();
 
 		public final SubredditPostListURL asSubredditPostListURL() {
-			return (SubredditPostListURL)this;
+			return (SubredditPostListURL) this;
 		}
 
 		public final SearchPostListURL asSearchPostListURL() {
-			return (SearchPostListURL)this;
+			return (SearchPostListURL) this;
 		}
 
 		public final UserPostListingURL asUserPostListURL() {
-			return (UserPostListingURL)this;
+			return (UserPostListingURL) this;
 		}
 
 		public UserProfileURL asUserProfileURL() {
-			return (UserProfileURL)this;
+			return (UserProfileURL) this;
```

Comment:
The existing coding style was to have no spaces after casts, or after for/if/catch/etc. If everyone who contributed reformatted the whole codebase, things would get very hectic ;)